server/portal/libs/agave/operations.py

```diff
@@ -253,34 +256,30 @@ def move(client, src_system, src_path, dest_system, dest_path, file_name=None):
 
     try:
         # list the directory and check if file_name exists
-        file_listing = client.files.list(systemId=dest_system, filePath=dest_path)
+        file_listing = client.files.listFiles(systemId=dest_system, path=dest_path)
         file_name = increment_file_name(listing=file_listing, file_name=file_name)
     except HTTPError as err:
         if err.response.status_code != 404:
             raise
 
-    full_dest_path = os.path.join(dest_path.strip('/'), file_name)
-
     if src_system == dest_system:
-        body = {'action': 'move',
-                'path': full_dest_path}
-        move_result = client.files.manage(systemId=src_system,
-                                          filePath=urllib.parse.quote(
-                                              src_path),
-                                          body=body)
-
-    if os.path.dirname(src_path) != dest_path or src_path != dest_path:
-        agave_indexer.apply_async(kwargs={'systemId': src_system,
-                                          'filePath': os.path.dirname(src_path),
-                                          'recurse': False},
-                                  routing_key='indexing')
-    agave_indexer.apply_async(kwargs={'systemId': dest_system,
-                                      'filePath': os.path.dirname(full_dest_path),
-                                      'recurse': False}, routing_key='indexing')
-    if move_result['nativeFormat'] == 'dir':
-        agave_indexer.apply_async(kwargs={'systemId': dest_system,
-                                          'filePath': full_dest_path, 'recurse': True},
-                                  routing_key='indexing')
+        move_result = client.files.moveCopy(systemId=src_system,
```

Comment:
Think we won't be able to test the `move` endpoint until #765 and #767

Reply:
The move command should be working. I was able to test it to move files within the system on `My Data (Work)` and `My Data (Frontera)`. Are you running into an error somewhere?

Reply:
@shayanaijaz "Move" is listing a different system
<img width="1139" alt="Screen Shot 2023-03-14 at 11 31 06 AM" src="https://user-images.githubusercontent.com/20326896/225073896-ca3240d2-ba81-4571-9d25-e752ec969500.png">
 from the system the action was initiated unfortunately